### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by atTheDisco** [DarkWav/SAC#88]
```
2018-10-27 [21:06:41] [Server thread/CRITICAL]: ErrorException: "Undefined property: DarkWav\SAC\Observer::$Player" (EXCEPTION) in "plugins/SAC_Source/src/DarkWav/SAC/Observer" at line 327
2018-10-27 [21:06:41] [Server thread/DEBUG]: #0 plugins/SAC_Source/src/DarkWav/SAC/Observer(327): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string Undefined property: DarkWav\SAC\Observer::$Player, string /root/server/plugins/SAC_Source/src/DarkWav/SAC/Observer.php, integer 327, array Array())
2018-10-27 [21:06:41] [Server thread/DEBUG]: #1 plugins/SAC_Source/src/DarkWav/SAC/Observer(1416): DarkWav\SAC\Observer->NotifyAdmins(string [SAC] > Esraoyunda is suspected for KillAura (VL = 24)!)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #2 plugins/SAC_Source/src/DarkWav/SAC/EventListener(171): DarkWav\SAC\Observer->PlayerHasDamaged(pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #3 src/pocketmine/plugin/MethodEventExecutor(38): DarkWav\SAC\EventListener->onDamage(pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #4 src/pocketmine/plugin/RegisteredListener(98): pocketmine\plugin\MethodEventExecutor->execute(DarkWav\SAC\EventListener object, pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #5 src/pocketmine/event/Event(96): pocketmine\plugin\RegisteredListener->callEvent(pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #6 src/pocketmine/entity/Entity(1169): pocketmine\event\Event->call()
2018-10-27 [21:06:41] [Server thread/DEBUG]: #7 src/pocketmine/entity/Living(645): pocketmine\entity\Entity->attack(pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #8 src/pocketmine/Player(3442): pocketmine\entity\Living->attack(pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #9 src/pocketmine/Player(2507): pocketmine\Player->attack(pocketmine\event\entity\EntityDamageByEntityEvent object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #10 src/pocketmine/network/mcpe/handler/SimpleSessionHandler(211): pocketmine\Player->attackEntity(pocketmine\Player object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #11 src/pocketmine/network/mcpe/protocol/InventoryTransactionPacket(155): pocketmine\network\mcpe\handler\SimpleSessionHandler->handleInventoryTransaction(pocketmine\network\mcpe\protocol\InventoryTransactionPacket object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #12 src/pocketmine/network/mcpe/NetworkSession(212): pocketmine\network\mcpe\protocol\InventoryTransactionPacket->handle(pocketmine\network\mcpe\handler\SimpleSessionHandler object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #13 src/pocketmine/network/mcpe/NetworkSession(192): pocketmine\network\mcpe\NetworkSession->handleDataPacket(pocketmine\network\mcpe\protocol\InventoryTransactionPacket object)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #14 src/pocketmine/network/mcpe/RakLibInterface(136): pocketmine\network\mcpe\NetworkSession->handleEncoded(string x.c.aRW.cf....4...............O.0........)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #15 vendor/pocketmine/raklib/src/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 94.54.244.70 3197, raklib\protocol\EncapsulatedPacket object, integer 0)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #16 src/pocketmine/network/mcpe/RakLibInterface(81): raklib\server\ServerHandler->handlePacket()
2018-10-27 [21:06:41] [Server thread/DEBUG]: #17 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
2018-10-27 [21:06:41] [Server thread/DEBUG]: #18 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
2018-10-27 [21:06:41] [Server thread/DEBUG]: #19 src/pocketmine/Server(2486): pocketmine\snooze\SleeperHandler->sleepUntil(double 1540674401.2431)
2018-10-27 [21:06:41] [Server thread/DEBUG]: #20 src/pocketmine/Server(2357): pocketmine\Server->tickProcessor()
2018-10-27 [21:06:41] [Server thread/DEBUG]: #21 src/pocketmine/Server(1866): pocketmine\Server->start()
```

**Comment by DarkWav** [comment]
Thanks for bug reporting! I will try to fix this ASAP!

**Comment by DarkWav** [comment]
I hope this is fixed aswell with SAC 3.6.3.
Make sure to upgrade to SAC 3.6.3 (Replace SAC_vx.x.x.phar).
If the bug still happens, please reply to this issue or open a new one.
